packages/commons/package.json

```diff
@@ -50,6 +51,7 @@
     "eslint": "8.57.0",
     "cpx2": "7.0.1",
     "prettier": "2.8.8",
-    "typescript": "5.4.5"
+    "typescript": "5.4.5",
+    "@types/nodemailer": "^6.4.15"
```

Comment:
Remove `^` so that dependency is pinned (after Renovate setup, we have only pinned dependencies)